src/stories/Gauge.js

```diff
@@ -20,6 +20,28 @@ const data = [
   }
 ]
 
-storiesOf('Gauge', module).addWithInfo('default', () => {
-  return <Gauge m={30} title='Ice Cream' data={data} />
-})
+const singleItem = [
+  {
+    value: 0,
+    name: 'Vanilla',
+    color: 'beige'
+  },
+  {
+    value: 0,
+    name: 'Chocolate',
+    color: 'brown'
+  },
+  {
+    value: 2,
+    name: 'Strawberry',
+    color: 'pink'
+  }
+]
+
+storiesOf('Gauge', module)
+  .addWithInfo('default', () => {
+    return <Gauge m={30} title='Ice Cream' data={data} />
+  })
+  .addWithInfo('single item', () => {
```

Comment:
Can we change this to `single segment`? 
`single item` makes it seem like there is only 1 item in the array.